static/js/modules/maps-event.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+var $ = require('jquery');
```

Comment:
The state map is also used in the elections page and doesn't have the map and table click events.